pmemcheck/tests/flush_check.stderr.exp

```diff
@@ -0,0 +1,14 @@
+Number of stores not made persistent: 1
+Stores not made persistent properly:
+[0]    at 0x........: main (flush_check.c:27)
+	Address: 0x........	size: 8	state: COMMITED
+Total memory not made persistent: 8
+
+Number of flushes on non-dirty stores: 3
+Stores flushed when not dirty:
```

Comment:
Actually, these are not "non-dirty stores". There were no stores, but only flushes. So, maybe we should rather report it as "Cache lines flushed when non-dirty".